kernels/volk/volk_32fc_accumulator_s32fc.h

```diff
@@ -0,0 +1,159 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2012, 2019 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * GNU Radio is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3, or (at your option)
+ * any later version.
+ *
+ * GNU Radio is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GNU Radio; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+/*!
+ * \page volk_32fc_accumulator_s32fc
```

Comment:
Volk kernels are traditionally named with the action being taken, not the object taking the action. So, not "accumulator", but rather "accumulate". Hence, please change the filename and all internal references to "accumulate".

Reply:
Done!

Reply:
well how about that! I somehow didn't see that kernel in my search ... you are totally correct! Let's just leave this one as "accumulator" to match the other one. That way there's no API change from past, just augmenting it. Resolving this request!